src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/stringliteralequality/InputStringLiteralEquality.java

```diff
@@ -39,7 +39,7 @@ void foo(String name)
             // what some of us get paid for :-)
         }
 
-        if ("Rick".toUpperCase(java.util.Locale.getDefault()) == "Rick".toLowerCase(java.util.Locale.getDefault()))
+        if ("Rick".toUpperCase(Locale.getDefault()) == "Rick".toLowerCase(Locale.getDefault()))
```

Comment:
Are we sure this test was testing a call on a full classpath?
Do we have another test case that does use the full classpath?
Why can't we just split the line up to keep the test case the same?